src/app/cluster/cluster-details/mla/alertmanager-config/component.ts

```diff
@@ -34,6 +35,11 @@ export enum Type {
   Grafana = 'Grafana',
 }
 
+export enum AddonType {
+  NodeExporter = 'node-exporter',
+  KubeStateMetrics = 'kube-state-metrics',
+}
```

Comment:
Is it certain that these two addons will be always available to install?

Reply:
Seems like they have to be installed manually during kkp installation.
MLA also needs some configuration and docs are explaining which addons you need to install https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/monitoring_logging_alerting/user_cluster/admin_guide/#addons-configuration

Do you think we should add some more information about these addons?

Reply:
I think yes. I'm not sure if everyone will be aware of that.

Reply:
@maciaszczykm added a warning only if addons aren't installable